docs/APIv1/accessibilityMap.yml

```
@@ -47,35 +47,35 @@ accessibilityMapRequest:
-      description: The maximum total travel time between origin and destination, including access, transfer and egress times
+      description: The maximum total travel time between origin and destination, including access, transfer and egress times. Defaults to 900 seconds (15 minutes).
       example: 900
     minWaitingTimeSeconds: 
       type: number
-      description: "The minimum time to wait at a stop/station, in seconds, to cope with uncertainties in the vehicle arrival times. Suggested value: 180"
+      description: "The minimum time to wait at a stop/station, in seconds, to cope with uncertainties in the vehicle arrival times. Suggested value: 180. Defaults to 180 seconds (3 minutes)."
       example: 180
     maxAccessEgressTravelTimeSeconds: 
       type: number
-      description:  Maximum time, in seconds, to reach the first stop/station in the trip, and to reach the destination from the last stop/station.
+      description:  Maximum time, in seconds, to reach the first stop/station in the trip, and to reach the destination from the last stop/station. Defaults to 900 seconds (15 minutes).
```

YAMLlint (1.38.0)

[error] too many spaces after colon

(colons)